app/out-of-office/server/scheduler.ts

```diff
@@ -0,0 +1,118 @@
+import { Meteor } from 'meteor/meteor';
+import Agenda from 'agenda';
+import { MongoInternals } from 'meteor/mongo';
+
+import { OutOfOfficeUsers } from '../../models/server';
+import { addToOutOfOfficeRoomsCollection, removeUserIdInPresentRooms } from './lib';
+
+interface IScheduleEnable {
+	userId: string;
+	when: string;
+	roomIds: string[];
+}
+
+interface IScheduleDisable {
+	userId: string;
+	when: string;
+}
+
+const EnableOutOfOfficeUserJob = 'enableOutOfOfficeUser';
+const DisableOutOfOfficeUserJob = 'disableOutOfOfficeUser';
+
+class OutOfOfficeScheduler {
+	private isConnected: boolean;
+
+	private scheduler: Agenda;
+
+	constructor() {
+		this.scheduler = new Agenda({
+			mongo: (
+				MongoInternals.defaultRemoteCollectionDriver().mongo as any
+			).client.db(),
+			db: { collection: 'outofoffice_scheduler' },
```

Comment:
please use namespaced collection names
```suggestion
			db: { collection: 'rocketchat_outofoffice_scheduler' },
```

Reply:
Renamed the collection to `rocketchat_outofoffice_scheduler`